src/test/java/net/technolords/micro/processor/ResponseProcessorTest.java

```diff
@@ -0,0 +1,61 @@
+package net.technolords.micro.processor;
+
+
+import junit.framework.Assert;
+import net.technolords.micro.config.ConfigurationManager;
+import org.apache.camel.Exchange;
+import org.apache.camel.impl.DefaultCamelContext;
+import org.apache.camel.impl.DefaultExchange;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class ResponseProcessorTest {
+    private final Logger LOGGER = LoggerFactory.getLogger(getClass());
+
+    String PATH_TO_CONFIG_FILE = "D:/commit2Learn/data/mockConfigurations/";
+    String PATH_TO_REQUEST = "D:/commit2Learn/data/mockRequests/";
+    String PATH_TO_RESPONSE = "D:/commit2Learn/data/mockResponses/";
```

Comment:
These path references are pointing to your local environment. As result, when I execute the test, or even do a
```
mvn clean install
```
it fails with:
```
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.38 sec <<< FAILURE! - in TestSuite
testPostResponses(net.technolords.micro.processor.ResponseProcessorTest)  Time elapsed: 0.008 sec  <<< FAILURE!
java.nio.file.NoSuchFileException: D:/commit2Learn/data/mockRequests/sample-post1-request.xml
        at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
        at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
        at java.nio.file.Files.newByteChannel(Files.java:361)
        at java.nio.file.Files.newByteChannel(Files.java:407)
        at java.nio.file.Files.readAllBytes(Files.java:3152)
        at net.technolords.micro.processor.ResponseProcessorTest.testPostResponses(ResponseProcessorTest.java:39)
```
In other words, you'd need to create a config file in src/test/resources just like you're doing with the test resources itself.